src/components/help/search/page/Page.js

```diff
@@ -43,18 +43,27 @@ export function Page({ result, isLoading, query }) {
   return (
     <React.Fragment>
       <Head>
-        <title>{pageTitle}</title>
-        <meta name="description" content={pageDescription}></meta>
-        <meta property="og:url" content={canonical.url} />
-        <meta property="og:type" content="website" />
-        <meta property="og:title" content={pageTitle} />
-        <meta property="og:description" content={pageDescription} />
+        <title key="title">{pageTitle}</title>
         <meta
+          key="description"
+          name="description"
+          content={pageDescription}
+        ></meta>
+        <meta key="og:url" property="og:url" content={canonical.url} />
+        <meta key="og:type" property="og:type" content="website" />
+        <meta key="og:title" property="og:title" content={pageTitle} />
+        <meta
+          key="og:description"
+          property="og:description"
+          content={pageDescription}
+        />
+        <meta
+          key="og:image"
```

Comment:
Vi burde lave en hjælper til det her. Det er præcis den samme mølle hver gang og de har jo været temmeligt besværligt for dig at rette det i alle de her filer...?

Reply:
Search and replace, så ikke så slemt, men ja det kunne vi godt, forhåbentlig er det ikke noget der skal røres så meget mere ved. key prop'en er en forglemmelse fra start. Men der skal ryddes ud i meta tagsne i de forskellige komponenter. Jeg tror at der er en del duplikationer. - Det ville også hjælpe på det :) - Jeg ville bare ikke begynde at gøre det i dette pull.